src/components/MapControls/MapControls.scss

```diff
@@ -37,3 +37,19 @@
   -webkit-transition: background-color 0.5s ease, color 0.5s ease;
   transition: background-color 0.5s ease, color 0.5s ease;
 }
+
+.tm-w-s {
+  .rs-zooms-bar {
+    .ol-zoomslider {
+      height: 150px;
+    }
+  }
+}
+
+.tm-w-xs {
+  .rs-zooms-bar {
+    .ol-zoomslider {
+      height: 120px;
+    }
+  }
+}
```

Comment:
the size of the zoom bar should be related to the  height not the width  `.tm-h-s`

Reply:
fixed